nextflow.config

```diff
@@ -5,3 +5,9 @@ process {
     '-A b2015110 --qos=short -p node'
   }
 }
+
+params {
+    programs {
```

Comment:
With the params block here I am able to pre-populate the parameters to the workflow.

I don't know exactly how the configuration file and the workflow script interacts. But I wanted somewhere to put paths to programs, these are things (along with slurm options), that the user of the workflow can change. I put it in params because then I know how to get it from within the workflow script, and it's not a bad thing if you can temporarily override from the command line, even if that was not my main intention.